tests/unit/variable_test.c

```diff
@@ -287,11 +287,13 @@ static void test_iterate_indices(void)
 
 // Below test relies on the ordering items in RB tree which is strongly
 // related to the hash function used.
+/* No more relevant, RBTree has been replaced with Map. */
+#if 0
```

Comment:
> It's there in the history if we need it.

Heh, famous last words. I'm quite sure that if in 6 months or more we'll reintroduce order in the variables and we need to test it, somebody will just rewrite a test if this is removed. Plus it's a good indication of functionality change.

I'm more in favour of keeping a bit of garbage in the test file, if it reminds everyone that opens it that something important changed. For some time at least.